src/components/combo-box/combo-box.ts

```diff
@@ -234,21 +231,6 @@ class BXComboBox extends BXDropdown {
     return `${prefix}-combo-box-item[selected]`;
   }
 
-  /**
-   * The name of the custom event fired before this combo box item is being toggled upon a user gesture.
-   * Cancellation of this event stops the user-initiated action of toggling this combo box item.
-   */
-  static get eventBeforeToggle() {
-    return `${prefix}-combo-box-beingtoggled`;
-  }
-
-  /**
-   * The name of the custom event fired after this combo box item is toggled upon a user gesture.
-   */
-  static get eventToggle() {
-    return `${prefix}-combo-box-toggled`;
-  }
-
```

Comment:
Any reason of removal...?

Reply:
Ah, I must've removed it by accident when pulling the changes from upstream. Will add it back!